content/en/docs/content-structure/layout-builder/_index.md

```diff
@@ -162,9 +199,35 @@ A title, headline, description, and link that usually display in the right or le
 | Label        | Machine Name | Type                                 | Required | Cardinality | Help text | Field Settings | Notes |
 |--------------|--------------|--------------------------------------|----------|-------------|-----------|----------------|-------|
 | Title        | field_title  | Text (plain)                         | yes      | 1           |
-| Body         | body         | Text (formatted, long, with summary) | no       | 1           | 
+| Body         | field_body         | Text (formatted, long, with summary) | no       | 1           |  |  | 
 | Icon / Image | field_image  | Entity reference                     | no       | 1           |           | Bundle: Image  |
-| CTA / Link   | field_cta    | Link                                 | no       | 1           |
+| CTA / Link   | field_link    | Link                                 | no       | 1           |
+
+## Related Articles
+
+- **Machine name:** `lb_related_articles`
+- **Project:** [lb_related_articles](<>)
+- **Designs:** [Mobile](<>) | [Desktop](<>)
+
+Component for displaying related articles within a page.
+
+| Label          | Machine Name                     | Type                       | Required | Cardinality | Help text | Field Settings         | Notes |
+|----------------|----------------------------------|----------------------------|----------|-------------|-----------|------------------------|-------|
+| Section heading     | (inherit)     |  |  |  |  |  | H2 |
+| Section subheading  | (inherit)     |  |  |  |  |  | H3 |
+
+## Related Events
```

Comment:
If everything about them otherwise is the same, I think `Related Articles` and `Related Events` could be covered under one block, either a more generalized `Views Reference` block or something like that.

Reply:
OK, sounds good! Whatever you think will make the most sense to developers :)

Reply:
I agree @froboy 
Unless there will be some not compatible differences in the future we don't know yet